doc/index.md

```diff
@@ -12,6 +12,35 @@ width: 125
 
 <h2 style="margin-top: 0.3em;">A high-level app and dashboarding solution for Python</h2>
 
+Panel is an [open-source and free](https://github.com/holoviz/panel/blob/main/LICENSE.txt) Python library that lets you create interactive tools, dashboards and data apps by connecting user-defined widgets to plots, images, tables, text and more.
```

Comment:
I've mainly moved the apps down. I'm in doubt here. Currently I hope users don't click on the apps to check them out. They are simply not of a high enough quality to showcase Panel for real.